src/client/app/avdelingsleder/bemanning/components/LeggTilSaksbehandlerForm.tsx

```diff
@@ -36,7 +36,7 @@ export const LeggTilSaksbehandlerForm: FunctionComponent = () => {
 			form.blur('epost');
 			return;
 		}
-		if (saksbehandlere.some((s) => s.epost.toLowerCase() === epost.toLowerCase())) {
+		if (!isLoadingSaksbehandlere && saksbehandlere.some((s) => s.epost.toLowerCase() === epost.toLowerCase())) {
```

Comment:
kanskje heller disable knappen for å legge til nye saksbehandlere til saksbehandlere er hentet ut